controllers/email-l10n.js

```diff
@@ -108,7 +110,59 @@ function notFound (req, res) {
   })
 }
 
+function previewEmail2022 (req, res) {
```

Comment:
FWIW, I did this to spot-check in the view:
```suggestion
function previewEmail2022 (req, res) {
  const unsubscribeUrl = `${AppConstants.SERVER_URL}/user/unsubscribe-monthly?token=${req.user.primary_verification_token}`
```

Reply:
Ah, I see – I tested actual emails and neglected the preview.  I will add this to avoid confusion 👍